packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -1069,6 +1072,9 @@ export class EntityInsertPanelImpl extends Component<Props, StateProps> {
 
             this.setSubmitting(false);
             onDataChange?.(false);
+
+            await incrementClientSideMetricCount(ENTITY_CREATION_METRIC, nounPlural + 'FileImport' + (isMerge ? 'WithMerge' : ''));
```

Comment:
We should make sure it's clear that the 'XFileImport' metric does NOT include the 'WithMerge', or else add a 'WithoutMerge" suffix for self-documentation.

Reply:
good call. adding the "WithoutMerge" suffix